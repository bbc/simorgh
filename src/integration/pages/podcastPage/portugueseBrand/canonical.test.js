/**
 * @service portuguese
 * @pathname /portuguese/podcasts/p07r3r3t
 */

import runCanonicalTests from '../canonicalTests';

describe('Canonical', () => {
  describe(pageType, () => {
    runCanonicalTests(service);
  });
});
