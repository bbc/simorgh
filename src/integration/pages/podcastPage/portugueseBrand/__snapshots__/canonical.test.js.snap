// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Podcast Page A11y I can see the skip to content link 1`] = `"Vá para o conteúdo"`;

exports[`Canonical Podcast Page Analytics ATI 1`] = `"<img height="1px" width="1px" alt="" style="position:absolute" src="https://logws1363.ati-host.net?s=598343&amp;s2=33&amp;p=portuguese.bbc_brasil.podcasts.podcasts.programmes.p07r3r3t.page&amp;x1=[urn%3Abbc%3Apips%3Ap07r3r3t]&amp;x2=[responsive]&amp;x3=[news-portuguese]&amp;x4=[pt-br]&amp;x7=[player-episode]&amp;x8=[simorgh-nojs]&amp;x9=[Que%2520Hist%C3%B3ria!%2520-%2520BBC%2520News%2520Brasil]"/>"`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 1`] = `
{
  "text": "BBC News, Brasil",
  "url": "/portuguese",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 2`] = `
{
  "text": "Por que você pode confiar na BBC",
  "url": "https://www.bbc.com/portuguese/institutional-50054434",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 3`] = `
{
  "text": "Termos de Uso",
  "url": "https://www.bbc.com/portuguese/institutional-36202448",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 4`] = `
{
  "text": "Sobre a BBC",
  "url": "https://www.bbc.com/portuguese/institutional-36202452",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 5`] = `
{
  "text": "Política de privacidade",
  "url": "https://www.bbc.com/portuguese/institutional-36202454",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 6`] = `
{
  "text": "Cookies",
  "url": "https://www.bbc.com/usingthebbc/cookies/",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 7`] = `
{
  "text": "Contate a BBC",
  "url": "https://www.bbc.co.uk/portuguese/send/u50853599",
}
`;

exports[`Canonical Podcast Page Footer Anchors should match text and url 8`] = `
{
  "text": "Leia sobre nossa política em relação a links externos.",
  "url": "https://www.bbc.co.uk/editorialguidelines/guidance/feeds-and-links",
}
`;

exports[`Canonical Podcast Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/portuguese"`;

exports[`Canonical Podcast Page Footer Paragraphs should match text 1`] = `"© 2024 BBC. A BBC não se responsabiliza pelo conteúdo de sites externos. Leia sobre nossa política em relação a links externos."`;

exports[`Canonical Podcast Page Header I can see the branding 1`] = `"BBC News, Brasil"`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 1`] = `
{
  "text": "Notícias",
  "url": "/portuguese",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 2`] = `
{
  "text": "Brasil",
  "url": "/portuguese/topics/cz74k717pw5t",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 3`] = `
{
  "text": "Eleições municipais",
  "url": "/portuguese/topics/cy6z19wz1zet",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 4`] = `
{
  "text": "Eleições EUA",
  "url": "/portuguese/topics/c30gn378n6kt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 5`] = `
{
  "text": "Internacional",
  "url": "/portuguese/topics/cmdm4ynm24kt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 6`] = `
{
  "text": "Economia",
  "url": "/portuguese/topics/cvjp2jr0k9rt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 7`] = `
{
  "text": "Saúde",
  "url": "/portuguese/topics/c340q430z4vt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 8`] = `
{
  "text": "Ciência",
  "url": "/portuguese/topics/cr50y580rjxt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 9`] = `
{
  "text": "Tecnologia",
  "url": "/portuguese/topics/c404v027pd4t",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 10`] = `
{
  "text": "Vídeos",
  "url": "/portuguese/topics/c9y2j35dn2zt",
}
`;

exports[`Canonical Podcast Page Header Navigation link should match text and url 11`] = `
{
  "text": "BBC Lê",
  "url": "/portuguese/topics/cxndrr1qgllt",
}
`;

exports[`Canonical Podcast Page I can see the brand title 1`] = `"Que História!"`;

exports[`Canonical Podcast Page I can see the episode summary 1`] = `"Andrea Fleck fez descoberta surpreendente após resolver pesquisar passado de seu pai, que tinha sido adotado quando bebê."`;

exports[`Canonical Podcast Page I can see the episode title 1`] = `"'Descobri que meu cabeleireiro é meu irmão'"`;

exports[`Canonical Podcast Page I can see the hero image 1`] = `"https://ichef.bbci.co.uk/images/ic/256x256/p09clsjd.jpg.webp"`;

exports[`Canonical Podcast Page Main heading should match text 1`] = `"Que História!, 'Descobri que meu cabeleireiro é meu irmão'"`;

exports[`Canonical Podcast Page Media Loader renders a valid container 1`] = `
<figure
  class="bbc-1qn0xuy"
  data-e2e="media-loader__container"
>
  <div
    class="bbc-1ty1x88"
    data-e2e="media-player"
  />
</figure>
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 1`] = `
{
  "sizes": null,
  "url": "http://localhost:7080/portuguese/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 2`] = `
{
  "sizes": "72x72",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-72x72.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 3`] = `
{
  "sizes": "96x96",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-96x96.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 4`] = `
{
  "sizes": "128x128",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-128x128.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 5`] = `
{
  "sizes": "144x144",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-144x144.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 6`] = `
{
  "sizes": "152x152",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-152x152.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 7`] = `
{
  "sizes": "180x180",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-180x180.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 8`] = `
{
  "sizes": "192x192",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 9`] = `
{
  "sizes": "384x384",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-384x384.png",
}
`;

exports[`Canonical Podcast Page SEO Apple Touch Icon should match attributes 10`] = `
{
  "sizes": "512x512",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/portuguese/images/icons/icon-512x512.png",
}
`;

exports[`Canonical Podcast Page SEO Canonical link 1`] = `"http://localhost:7080/portuguese/podcasts/p07r3r3t"`;

exports[`Canonical Podcast Page SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical Podcast Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Podcast Page SEO Heading level 1 1`] = `"Que História!, 'Descobri que meu cabeleireiro é meu irmão'"`;

exports[`Canonical Podcast Page SEO Lang attribute 1`] = `"pt-br"`;

exports[`Canonical Podcast Page SEO Linked data should match text 1`] = `
{
  "@context": "http://schema.org",
  "@graph": [
    {
      "@type": "WebPage",
      "image": {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
        "width": 1024,
      },
      "inLanguage": {
        "@type": "Language",
        "alternateName": "pt-BR",
        "name": "Portuguese",
      },
      "mainEntityOfPage": {
        "@id": "http://localhost:7080/portuguese/podcasts/p07r3r3t",
        "@type": "WebPage",
        "name": "'Descobri que meu cabeleireiro é meu irmão' - Que História! - News Brasil",
      },
      "publisher": {
        "@type": "NewsMediaOrganization",
        "logo": {
          "@type": "ImageObject",
          "height": 576,
          "url": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
          "width": 1024,
        },
        "name": "BBC News Brasil",
        "publishingPrinciples": "https://www.bbc.com/portuguese/institutional-50054434",
      },
      "thumbnailUrl": "https://news.files.bbci.co.uk/include/articles/public/portuguese/images/metadata/poster-1024x576.png",
      "url": "http://localhost:7080/portuguese/podcasts/p07r3r3t",
    },
    {
      "@type": "AudioObject",
      "description": "Andrea e Troy se conheceram no salão e resolveram pesquisar suas origens.",
      "duration": "PT12M40S",
      "name": "Que História!",
      "thumbnailUrl": "https://ichef.bbci.co.uk/images/ic/1024x576/p09clsjd.jpg",
      "uploadDate": "2021-06-18T00:00:00.000Z",
    },
  ],
}
`;

exports[`Canonical Podcast Page SEO OG description 1`] = `"Andrea e Troy se conheceram no salão e resolveram pesquisar suas origens."`;

exports[`Canonical Podcast Page SEO OG image 1`] = `"https://ichef.bbci.co.uk/images/ic/400x400/p09clsjd.jpg"`;

exports[`Canonical Podcast Page SEO OG image alt 1`] = `"BBC News Brasil"`;

exports[`Canonical Podcast Page SEO OG locale 1`] = `"pt-BR"`;

exports[`Canonical Podcast Page SEO OG site name 1`] = `"BBC News Brasil"`;

exports[`Canonical Podcast Page SEO OG title 1`] = `"'Descobri que meu cabeleireiro é meu irmão' - Que História! - News Brasil - BBC News Brasil"`;

exports[`Canonical Podcast Page SEO OG type 1`] = `"website"`;

exports[`Canonical Podcast Page SEO OG url 1`] = `"http://localhost:7080/portuguese/podcasts/p07r3r3t"`;

exports[`Canonical Podcast Page SEO Page title 1`] = `"'Descobri que meu cabeleireiro é meu irmão' - Que História! - News Brasil - BBC News Brasil"`;

exports[`Canonical Podcast Page SEO Robots meta tag 1`] = `"noodp, noydir, max-image-preview:large"`;

exports[`Canonical Podcast Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Podcast Page SEO Twitter creator 1`] = `"@bbcbrasil"`;

exports[`Canonical Podcast Page SEO Twitter description 1`] = `"Andrea e Troy se conheceram no salão e resolveram pesquisar suas origens."`;

exports[`Canonical Podcast Page SEO Twitter image alt 1`] = `"BBC News Brasil"`;

exports[`Canonical Podcast Page SEO Twitter image src 1`] = `"https://ichef.bbci.co.uk/images/ic/400x400/p09clsjd.jpg"`;

exports[`Canonical Podcast Page SEO Twitter site 1`] = `"@bbcbrasil"`;

exports[`Canonical Podcast Page SEO Twitter title 1`] = `"'Descobri que meu cabeleireiro é meu irmão' - Que História! - News Brasil - BBC News Brasil"`;

exports[`Canonical Podcast Page a11y Assistive technology reads the brand and episode title as the headline 1`] = `"Que História!, 'Descobri que meu cabeleireiro é meu irmão'"`;
